src/encoded/schemas/enzyme.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Enzyme",
-    "description": "DNA digestion enzymes like HindIII or DNaseI",
+    "description": "DNA restriction enzymes like HindIII or DNaseI",
```

Comment:
Why the change from digestion to restriction? Micrococcal nuclease isn't a restriction enzyme, for example.

Reply:
Oh, now I see why it was there in the first place. Then endonuclease is what we mean. Digestion enzyme didn't sound right, but I understand now. I reverted that commit.